BBallStatsTrack/src/com/example/bballstatstrack/adapter/GameAdapter.java

```diff
@@ -0,0 +1,52 @@
+package com.example.bballstatstrack.adapter;
+
+import java.util.ArrayList;
+
+import com.example.bballstatstrack.R;
+import com.example.bballstatstrack.models.Game;
+
+import android.app.Activity;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+public class GameAdapter extends ArrayAdapter< Game >
+{
+    Activity mActivity;
+
+    public GameAdapter( ArrayList< Game > games, Activity activity )
+    {
+        super( activity, android.R.layout.simple_list_item_1, games );
+        mActivity = activity;
+    }
+
+    @Override
+    public View getView( int position, View convertView, ViewGroup parent )
+    {
+        if( null == convertView )
+        {
+            convertView = mActivity.getLayoutInflater().inflate( R.layout.list_games, null );
```

Comment:
just to note po you can also use
LayoutInflater.from(parent.getContext());
To get the inflater instead of asking for an activity in the constructor just for the sake of obtaining the LayoutInflater po :bow: